shared/components/BaseButton/BaseButton.modules.scss

```diff
@@ -50,6 +50,7 @@ $button-text-color: $color-white;
 .primary {
   background-color: $primary-bg-color;
   color: $button-text-color;
+  display: inline-block;
```

Comment:
Please move this to the `.sizing` class selector, that way all variants inherit the change. This needs to be tested cross-browser as well, since `inline-flex` was added to fix an issue in IE11, See #793